python/doc/examples/functional_modeling/quick_start_functions.ipynb

```diff
@@ -0,0 +1,585 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Defining Python and symbolic functions: a quick start introduction to functions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Abstract\n",
+    "\n",
+    "In this example, we show how to define Python and symbolic functions. This allows to define functions that the library can evaluate, so that it can be used, for example, to propagate the uncertainties. We focus on functions which have a vector input and a vector output."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "We consider the following example:\n",
+    "* three input variables, \n",
+    "* two outputs. \n",
+    "\n",
+    "Moreover, we assume that the input distribution has independent gaussian marginals. \n",
+    "\n",
+    "The function is defined by the equations:\n",
+    "$$\n",
+    "Y_1 = X_1 + X_2 + X_3\n",
+    "$$\n",
+    "\n",
+    "and \n",
+    "\n",
+    "$$\n",
+    "Y_2 = X_1 - X_2 X_3\n",
+    "$$\n",
+    "\n",
+    "for any $X_1,X_2,X_3\\mathbb{R}$. \n",
+    "\n",
+    "The exact expectation and standard deviation of the output random variable are presented in the following table.\n",
+    "\n",
+    "| Variable | Expectation | Standard deviation |\n",
+    "|-|-|-|\n",
+    "| $Y_1$ | 0 | 1.732 |\n",
+    "| $Y_2$ | 0 | 1.415 |\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openturns as ot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We first define the input random vector of the function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X0 = ot.Normal(0.,1.)\n",
+    "X1 = ot.Normal(0.,1.)\n",
+    "X2 = ot.Normal(0.,1.)\n",
+    "inputDistribution = ot.ComposedDistribution((X0,X1,X2))\n",
+    "inputRandomVector = ot.RandomVector(inputDistribution)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Python function\n",
+    "\n",
+    "Based on a Python function defined with the `def` keyword, the `PythonFunction` class creates a function. \n",
+    "\n",
+    "The simplest constructor of the `PythonFunction` class is:\n",
+    "\n",
+    "`PythonFunction ( nbInputs , nbOutputs , myPythonFunc )`\n",
+    "\n",
+    "where\n",
+    "\n",
+    "* `nbInputs`: the number of inputs, \n",
+    "* `nbOutputs`: the number of outputs,\n",
+    "* `myPythonFunc`: a Python function.\n",
+    "\n",
+    "The goal of the `PythonFunction` class are:\n",
+    "\n",
+    "* to easily create a function in Python,\n",
+    "* use all the power of the Python libraries in order to evaluate the function, including, if required, powerful librairies. "
```

Comment:
```suggestion
    "* use all the power of the Python libraries in order to evaluate the function, including, if required, powerful libraries. "
```
In case you discard the previous suggestion, you should correct the typo *libra**i**ries*.

Reply:
I never discard!